source/lib/member-stack.test.ts

```diff
@@ -439,6 +433,19 @@ describe('member stack', function () {
     return { startingNodes, remainingNodes };
   }
 
+  function getConditionalDependencyLogicalIds(gate: Resource, stacks: { [_: string]: Resource }): string[] {
+    const result: string[] = [];
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    for (const [_, value] of Object.entries(gate.Metadata)) {
+      const metadata = value as any; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
Does this have to be `any`? could it be narrowed down to an interface in cdk such as [MetadataEntry](https://docs.aws.amazon.com/cdk/api/v2/docs/constructs.MetadataEntry.html)